src/playwright/matchers/to-be-accessible.ts

```diff
@@ -2,23 +2,13 @@ import { type Page, expect } from '@playwright/test';
 import { AxeBuilder } from '@axe-core/playwright';
 
 export default expect.extend({
-  /**
-   * @param expectedViolations - An array of Rule IDs¹.
-   *
-   * 1: https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md
-   */
```

Comment:
Moved to `test.ts` so it shows up on hover:

<img width="926" height="220" alt="image" src="https://github.com/user-attachments/assets/44c23b97-6037-4e3b-b1fe-6765ac0706ba" />